client/my-sites/domains/domain-management/domain-connect-mapping/index.jsx

```diff
@@ -186,12 +184,13 @@ class DomainConnectMapping extends Component {
 			'?' +
 			this.getRedirectUriStatusString();
 
-		wpcom
-			.getDomainConnectSyncUxUrl(
-				this.props.selectedDomainName,
-				'WordPress.com',
-				'hosting',
-				redirectUri
+		const providerId = 'WordPress.com';
+		const serviceId = 'hosting';
+
+		wp.req
+			.get(
+				`/domains/${ this.props.selectedDomainName }/dns/providers/${ providerId }/services/${ serviceId }/syncurl`,
+				{ redirect_uri: redirectUri }
```

Comment:
Can you inline the provider and service IDs into the URL?

Also, destructuring the `selectedDomainName` prop at the beginning would make the entire method quite shorter.

Reply:
I agree - addressed in 80be30750ce72e2bfda5138aabcb0bd2de9c9ba5 and 10e36defa1a9fbaf90a1ec8ce94e5651bfb2727e.